core/src/lib.rs

```diff
@@ -23,17 +23,26 @@
 //! The main concepts of libp2p-core are:
 //!
 //! - A [`PeerId`] is a unique global identifier for a node on the network.
-//!   Each node must have a different `PeerId`. Normally, a `PeerId` is the
+//!   Each node must have a different [`PeerId`]. Normally, a [`PeerId`] is the
```

Comment:
It is often intentional that not every mentioning of a (linkable) name is actually a link. Specifically, it is a common practice to only link the first occurrence of such a name in a single paragraph or doc section, using `regular monospaced text` for subsequent occurrences. Too many links in a block of text, especially for the same name, can be quite irritating.